<p>This rule detects hardcoding an IP address which should not be done in cloud-ready code.</p>
<h2>Noncompliant Code Example</h2>
<pre>
String ip = "192.168.1.111";
</pre>
<h2>Compliant Solution</h2>
<pre>
TO DO 
</pre>
