<p>This rule detects writing to a local file which should not be done in cloud-ready code.</p>
<h2>Noncompliant Code Example</h2>
<pre>
File x = new File("C://Documents/Temp.txt"); // Open local file
</pre>
<h2>Compliant Solution</h2>
<pre>
Connection conn = DriverManager.getConnection(/* ... */); // Open database connection
</pre>
